src/sortedDict.jl

```diff
@@ -256,22 +260,55 @@ immutable SDKeyIteration{T <: SDIterableTypesBase}
     base::T
 end
 
+eltype{K,D,Ord <: Ordering}(s::SDKeyIteration{SortedDict{K,D,Ord}}) = K
+eltype{K,D,Ord <: Ordering}(s::SDKeyIteration{ExcludeLast{K,D,Ord}}) = K
+eltype{K,D,Ord <: Ordering}(s::SDKeyIteration{IncludeLast{K,D,Ord}}) = K
```

Comment:
I think this (and most of the ones below) cannot be simplified for the same reason `Array{Complex}` won't match `Array{Complex128}`